tests/unit/test_util.py

```diff
@@ -10,8 +10,12 @@
 import unittest
 from unittest.mock import patch
 from io import StringIO
+from tempfile import TemporaryDirectory
 
+def test_function(arg):
```

Comment:
I think this is not being used anywhere, or is it?

Reply:
Oh yes, now it is not being used, removing it in next commit.